### PR TITLE
fix: handling of empty table cells

### PR DESCRIPTION
**Pull request by kleinfreund**
**fix(zones): handling of empty table cells**

Fixes empty cells not rendering the right content.

Fixes the warnings cell showing `false` instead of nothing.

**fix(services): handling of empty table cells**

Randomizes presence of certain fields on service mocks and only sets dataplanes if the service is an external one.

Makes sure to always show a status badge.

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

File: src/app/services/views/ServiceListView.vue [hunk 1]
```diff
@@ -65,14 +85,7 @@
...
               <template #status="{ row: item }">
-                <StatusBadge
-                  v-if="item.status"
-                  :status="item.status"
-                />
-
-                <template v-else>
-                  —
-                </template>
+                <StatusBadge :status="item.status || 'not_available'" />
```
Comment (lahabana) [inline src/app/services/views/ServiceListView.vue:88]: Would it make sense to have this done in your `transfromToTableData` ?

Comment (kleinfreund) [reply src/app/services/views/ServiceListView.vue:88]: Technically, yes, but it looks very much like this (these transform call) is all going to be thrown out in favor of a different approach.

Comment (lahabana) [reply src/app/services/views/ServiceListView.vue:88]: Yes but the transform call will likely be the base for the code of the new approach no?

File: src/app/zones/views/ZoneListView.vue [hunk 2]
```diff
@@ -72,11 +72,23 @@
...
 
                 <template #zoneCpVersion="{ rowValue }">
-                  {{ rowValue }}
+                  <template v-if="rowValue">
+                    {{ rowValue }}
+                  </template>
+
+                  <template v-else>
+                    —
+                  </template>
```
[the remark this answers is not in the document]
Comment (kleinfreund) [reply src/app/zones/views/ZoneListView.vue]: That works, too. I’ve changed the instance in the two list views I’m changing in this PR.

**Review by lahabana — APPROVED** [review]

**Pull request merged**